cli/src/commands/grpc-service/commands/generate.ts

```diff
@@ -30,20 +50,55 @@ export default (opts: BaseCommandOptions) => {
   command.option('-o, --output <path-to-output>', 'The output directory for the protobuf schema. (default ".").', '.');
   command.option('-p, --package-name <name>', 'The name of the proto package. (default "service.v1")', 'service.v1');
   command.option('-g, --go-package <name>', 'Adds an `option go_package` to the proto file.');
+  // NOTE: The following language-specific options are not enabled for the alpha release
+  // command.option('--java-package <name>', 'Adds an `option java_package` to the proto file.');
+  // command.option('--java-outer-classname <name>', 'Adds an `option java_outer_classname` to the proto file.');
+  // command.option('--java-multiple-files', 'Adds `option java_multiple_files = true` to the proto file.');
+  // command.option('--csharp-namespace <name>', 'Adds an `option csharp_namespace` to the proto file.');
+  // command.option('--ruby-package <name>', 'Adds an `option ruby_package` to the proto file.');
+  // command.option('--php-namespace <name>', 'Adds an `option php_namespace` to the proto file.');
+  // command.option('--php-metadata-namespace <name>', 'Adds an `option php_metadata_namespace` to the proto file.');
+  // command.option('--objc-class-prefix <name>', 'Adds an `option objc_class_prefix` to the proto file.');
+  // command.option('--swift-prefix <name>', 'Adds an `option swift_prefix` to the proto file.');
   command.option(
     '-l, --proto-lock <path-to-proto-lock>',
     'The path to the existing proto lock file to use as the starting point for the updated proto lock file. ' +
       'Default is to use and overwrite the output file "<outdir>/service.proto.lock.json".',
   );
+  command.option(
+    '-w, --with-operations <path-to-operations>',
+    'Path to directory containing GraphQL operation files (.graphql, .gql, .graphqls, .gqls). ' +
+      'When provided, generates proto from operations instead of SDL types.',
```

Comment:
It doesn't accept pattern or work recursively eight?

Reply:
Updated